platform-core-module/src/main/java/com/softicar/platform/core/module/file/stored/StoredFileConverter.java

```diff
@@ -0,0 +1,44 @@
+package com.softicar.platform.core.module.file.stored;
+
+import com.softicar.platform.common.io.mime.MimeType;
+import com.softicar.platform.core.module.email.converter.EmailToPdfConverter;
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * Facilitates conversion of {@link AGStoredFile} instances.
+ *
+ * @author Alexander Schmidt
+ */
+public class StoredFileConverter {
```

Comment:
hmm, this API makes it hard to find it...I suggest to make this class package-private and add a `AGStoredFile.toPdf` method that we can then extend to support more and more formats in the future.

Reply:
Did something similar. See `StoredFileEmailPreviewPopup` as an example:
`file.convert().fromEmailToPdfBytes().ifPresentOrElse(this::appendPdfDisplay, this::appendFailureMessage);`